### PR TITLE
Fix GET body in balance operations and bulk anticipations

### PR DESCRIPTION
**Pull request by LeoNero**
## Description
This PR fix some small problems related to query parameters

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

**Comment by GitCop** [comment]
We found issues in commit naming convention

 * Commit: 74873dfff1009d2dfed0bcfb12261e7104547097
  * Bons títulos tem no máximo até 50 caracteres. Para adicionar mais detalhes use o corpo!

O guia completo de estilo de commits está disponível em https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
We found issues in commit naming convention

 * Commit: 74873dfff1009d2dfed0bcfb12261e7104547097
  * Bons títulos tem no máximo até 50 caracteres. Para adicionar mais detalhes use o corpo!

O guia completo de estilo de commits está disponível em https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: lib/resources/bulkAnticipations.js [hunk 1]
```diff
@@ -17,7 +23,11 @@ const findOne = curry((opts, body) =>
 )
 
 const findAll = curry((opts, body) =>
-  request.get(opts, routes.bulkAnticipations.base(body.recipientId), body)
+  request.get(
+    opts,
+    routes.bulkAnticipations.base(body.recipientId),
+    dissoc('recipientId', body)
```
Comment (MarcoWorms) [inline lib/resources/bulkAnticipations.js]: @LeoNero pq rola esse dissoc aqui? Só queria entender pq não tem explicado isso no PR :)

Comment (derekstavis) [reply lib/resources/bulkAnticipations.js]: a gente aceita um `recipientId` como payload dessa call, porém usamos ele na URL, vs o restante que vai pro corpo.

**Pull request merged**